package.json

```diff
@@ -2,7 +2,7 @@
   "name": "notifications-panel",
   "version": "1.2.1",
```

Comment:
this is probably worth a major version bump, no? since software already using this would have to update in order not to break?